CHANGELOG.md

```diff
@@ -1,7 +1,16 @@
 ### citus v11.0.2 (June 15, 2022) ###
 
+* Drops support for PostgreSQL 12
+
 * Open sources enterprise features, see the rest of changelog items
 
+* Turns metadata syncing on by default
+
+* Adds `citus_finalize_upgrade_to_citus11()` which is necessary to upgrade to
+  Citus 11+ from earlier versions
+
+* Introduces `citus_finish_citus_upgrade()` procedure
+
 * Open sources non-blocking shard moves/shard rebalancer
```

Comment:
`Open sources blocking tenant_isolation`  -> I think we can drop `blocking` because we only have 1 type of tenant isolation ?